wsd/DocumentBroker.cpp

```diff
@@ -4401,11 +4401,14 @@ void DocumentBroker::handleTileCombinedRequest(TileCombined& tileCombined, bool
         bool tooLarge = cachedTile && cachedTile->tooLarge();
         if(!cachedTile || !cachedTile->isValid() || tooLarge)
         {
+            bool forceKeyFrame = false;
             if (!cachedTile || tooLarge)
+            {
+                forceKeyFrame = true;
```

Comment:
Looks sensible to me =) I was worried about the race between the kit and coolwsd but we request this tile anyway I guess.